rts/lra/test/tck/pom.xml

```diff
@@ -38,14 +32,7 @@
                         <include>**/*Test*.java</include>
                     </includes>
                     <systemPropertyVariables combine.children="append">
-                        <java.util.logging.config.file>${project.build.testOutputDirectory}/logging.properties</java.util.logging.config.file>
-                        <lra.tck.suite.host>${lra.tck.suite.host}</lra.tck.suite.host>
-                        <lra.tck.suite.port>${lra.tck.suite.port}</lra.tck.suite.port>
-                        <thorntail.test.logging.category.arjuna>${thorntail.test.logging.category.arjuna}</thorntail.test.logging.category.arjuna>
-                        <thorntail.test.logging.category.narayana>${thorntail.test.logging.category.narayana}</thorntail.test.logging.category.narayana>
-                        <lra.coordinator.port>${lra.coordinator.port}</lra.coordinator.port>
-                        <lra.tck.suite.debug.params>${lra.tck.suite.debug.params}</lra.tck.suite.debug.params>
-                        <lra.tck.timeout.factor>${lra.test.timeout.factor}</lra.tck.timeout.factor>
+                        <lra.tck.timeout.factor>${lra.tck.timeout.factor}</lra.tck.timeout.factor>
```

Comment:
This is where the TCK factor is not being set by default making the timelimit tests fail.

Reply:
See my PR review summary - the build was failing unless I pass `-Dlra.tck.timeout.factor=num` to the build.

Reply:
+1, thanks for pointing that out. The scripts were wrong and I took the wrong property. I hope it will be ok now.